### PR TITLE
MoveLightningAbility `offsetX`

### PR DESCRIPTION
**Pull request by MEEPofFaith**
Adds a horizontal offset to move lightning ability.

Hey @BlueWolf3682 I finally did what you requested.

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by Anuken** [comment]
This breaks compatibility, don't add it to the constructor.

> I have ensured that any new features in this PR function correctly in-game, if applicable.

You should.

**Comment by MEEPofFaith** [comment]
Happens when I don't have the time, but now I do.

**Comment by MEEPofFaith** [comment]
I see what you mean now

**Comment by MEEPofFaith** [comment]
I can't tell why it doesn't spawn lightning, Log.info isn't giving me null errors.

**Comment by MEEPofFaith** [comment]
Oh wait I think I still have effects turned off

**Comment by MEEPofFaith** [comment]
Made it compatable with previous versions.

`offset` shifts on the Y axis
`offsetX` shifts on the X axis

**Comment by MEEPofFaith** [comment]
https://user-images.githubusercontent.com/54301439/128621102-72184afb-f5a8-4b13-a5d1-617ad1c71a2a.mp4

File: core/src/mindustry/entities/abilities/MoveLightningAbility.java [hunk 1]
```diff
@@ -34,23 +34,27 @@
     public Effect shootEffect = Fx.sparkShoot;
     public Sound shootSound = Sounds.spark;
     
+    protected float side = 1f;
+    
     MoveLightningAbility(){}
     
-    public MoveLightningAbility(float damage, int length, float chance, float offset, float minSpeed, float maxSpeed, Color color, String heatRegion){
+    public MoveLightningAbility(float damage, int length, float chance, float offsetX, float offset, float minSpeed, float maxSpeed, Color color, String heatRegion){
```
Comment (Anuken) [inline core/src/mindustry/entities/abilities/MoveLightningAbility.java:41]: Again, don't add it to the constructors! This breaks compatibility.

Comment (MEEPofFaith) [reply core/src/mindustry/entities/abilities/MoveLightningAbility.java:41]: Oh wait, now I get what you mean

**Comment by MEEPofFaith** [comment]
nice

**Pull request closed without merging**